lib/Optimizer/GraphOptimizer/GraphOptimizer.cpp

```diff
@@ -2864,7 +2864,8 @@ bool FoldChannelShuffle::run(Function *F, const CompilationContext &cctx) {
 void glow::fold(Function *F, CompilationContext &cctx) {
   LOG_SCOPE(F->getLogContext(), "glow::fold")
 
-  FunctionPassManager FPM("FoldFPM", createDefaultFoldPassPipeline());
+  FunctionPassManager FPM("FoldFPM", createDefaultFoldPassPipeline(),
+                          nullptr /*backend*/);
```

Comment:
nit: Since nullptr is already the default, do we need to explicitly pass it? I'd say either don't pass it, or remove nullptr as the default.